torch/fx/experimental/proxy_tensor.py

```diff
@@ -633,6 +548,8 @@ def dispatch_trace(
         concrete_args: Optional[Tuple[Any, ...]] = None,
 ) -> GraphModule:
     graph = tracer.trace(root, concrete_args)
+    from torch._inductor.fx_passes.dedupe_symint_uses import dedupe_symints
+    dedupe_symints(graph)
```

Comment:
oh you want to unconditionally call it here? I guess we can...

Reply:
Right, that was the previous state